doc/index.rst

```diff
@@ -22,6 +22,7 @@ Developer documentation
    provides
    templates
    taxes
+   addons
```

Comment:
Addons and provides are somewhat connected. IMHO addons should be added before provides into that list. What do you think?